### PR TITLE
owai, langley and ayshe plushies

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

Adds owai, langley and ayshe plushies

## Why It's Good For The Game

plushies are nice

## Changelog
:cl:
add: owai, langley and ayshe plushies
/:cl:

### CODE REVIEW

**Review by Ghommie — CHANGES_REQUESTED** [review]
Stop this at once, I can't take any of it anymore. Do I really have to pull off a feature freeze for plushies? Like, seriously?

**Comment by Seris02** [comment]
May I just ask...
what the fuck is so wrong about plushies.
people like having plushies of themselves ingame, they sprite them or have someone else do it.
why do you care so much about this that you add "Do Not Merge" to a PR about plushies.
If you hate them this much, I'd like a detailed and _valid_ reason for why.

**Comment by CameronWoof** [comment]
Seconding the request for a reason other than personal ones.

**Comment by deathride58** [comment]
More plushies are fine.

**Comment by Ghommie** [comment]
No, I'm fed up of people justifying this stupidity as something innocuous, and I'm referring to the character reference plushes. I have nothing against the "generic" ones (such as nuke op, lamp, bee).

Yet people only want this bandwagon to go on so they can stuff more of their personas as time goes, and by doing so they encourage future contributors to do the same.
I don't know more than half these characters, and a good chunk of the rest is no longer around either, and without said characters the plushes don't even have a meaning, they are just ugly sprites and palette swaps.

TL;DR: We have enough unmemorable lame plushes. Stop thinking of these as mementos of your characters and contribute real stuff instead.

**Comment by silicons** [comment]
this is honestly fine if silly
that said why isn't this a server side config file yet lol

**Comment by BlackMajor** [comment]
Kinda yikes up in here.

**Comment by Poojawa** [comment]
Ghommie's just jealous he doesn't have a static worth being a plushie toy for.

**Comment by Ghommie** [comment]
Nah, I just never cared to get one. It feels pretty stupid.

**Comment by Ghommie** [comment]
Fairly confident this will be merged on the aforementioned PR at some point anyway.

**Comment by AysheDaArtist** [comment]
Holy moly, this was only 3-4 years ago?  Wild :P

**Comment by BlackMajor** [comment]
> Holy moly, this was only 3-4 years ago? Wild :P

Why would you Necro this

**Comment by CameronWoof** [comment]
> > Holy moly, this was only 3-4 years ago? Wild :P
> 
> Why would you Necro this

Because of e-mail notifications, I'm here, too! Miss you, bro.

**Comment by Ghommie** [comment]
Looking back, this was embarassingly callous of me, immature even, though I wasn't fond of the idea of shoving cameos of cringy (just kidding, just kidding!) OCs into the code. At least it's water under the bridge now. :)

EDIT: Also yeah, don't necro it. 😬
